### DISCUSSION TITLE
사소한 부분이기는 한데 +_+;; ERD field 이름 중 투표 필드 이름 voteSpyCount 어떨까요?

### DISCUSSION THREAD

**Discussion by ghost** [tastekim/WeAllLie-BE#18] [Ideas]
오전에 스키마 작성하려고 노션 가서 ERD 봤는데 
spyPlayCount
spyWinCount
이거 너무 직관적이고 좋은 것 같아요 !!! :+1::+1:

그거 보고 생각났는데 
voteCount도 voteSpyCount 로 하면 어떨까요?

**Comment by tastekim** [comment]
좋슴니다 ! 스파이를 투표하는 내용이니 좀 더 직관적이겠네요 !
